<launch>
    <param    name ="/ros_speech2text/audio_rate"      value="44100" />
    <param    name ="/ros_speech2text/speech_history"  value="~/.ros/ros_speech2text/speech_history" />
    <!-- param    name ="/ros_speech2text/api_credentials" value="$(find ros_speech2text)/GCloud_SpeechAPI_Cred" /-->
    <env      name ="GOOGLE_APPLICATION_CREDENTIALS"   value="$(find ros_speech2text)/GCloud_SpeechAPI_Cred" />
    <param    name ="/ros_speech2text/audio_threshold" value="700"   />
    <param 	  name ="/ros_speech2text/audio_device_idx" value="0"  />
    <param    name ="/ros_speech2text/enable_dynamic_threshold" value="True"  />
    <param    name ="/ros_speech2text/audio_dynamic_percentage" value="50"  />
    <param    name ="/ros_speech2text/audio_dynamic_frame" value="3"  />

    <rosparam param="/ros_speech2text/speech_context">
        ["screwdriver","left","right","yes","no","baxter","screw","table","hammer","board","can","you","tuck","untuck","give","pass","hand","move","help"]
    </rosparam>

    <node pkg="ros_speech2text" name="ros_speech2text" type="ros_speech2text_sync.py" output="screen" />

</launch>
